src/systemtest/java/reposense/ConfigSystemTest.java

```diff
@@ -105,8 +107,11 @@ private void generateReport(String inputDates)
                 new RepoConfigCsvParser(((ConfigCliArguments) cliArguments).getRepoConfigFilePath()).parse();
         List<AuthorConfiguration> authorConfigs =
                 new AuthorConfigCsvParser(((ConfigCliArguments) cliArguments).getAuthorConfigFilePath()).parse();
+        List<GroupConfiguration> groupConfigs =
+                new GroupConfigCsvParser(((ConfigCliArguments) cliArguments).getGroupConfigFilePath()).parse();
```

Comment:
The way you use the parser does not seems to be optional? (Not sure)

Reply:
Erm.. We just parse in the `groupConfigs` here in the systems test. The groupConfigs in `RepoSense#getRepoConfigurations` is optional if the user does not want to supply custom groupings.

Reply:
OK, I saw your previous comment of why can not combine the try together.